### ISSUE TITLE
Add `script.callFunction` command

### ISSUE THREAD

**Issue by sadym-chromium** [w3c/webdriver-bidi#140]
As a BiDi user, I want to be able to invoke some custom function with some custom arguments on the page context.
For example, I want to be able to invoke `(text) => {this.innerHtml=text}` with custom `this` and `text`.

It can be done:
1. As a part of the `script.evaluate` command by adding optional parameters `thisObject` and `argument`. It can be quite ugly, as `expression` changes meaning from **expression to be evaluated and returned as a result** to **expression to return a function which will be invoked with the given params and each result will be returned to user**.
2. As another command `script.invoke` with an `expression` string returning function to be invoked, `thisObject` and `argument`.

I'm inclined to go with the second way.

There is another open question:
1. How should the arguments be presented? As a user, I want to be ale to use both some constants, simple objects like `{a: 1}`, and `RemoteObject` return by e.g. `script.evaluate`.

For primitive type it is logical to use the [`remote value`](https://w3c.github.io/webdriver-bidi/#type-common-RemoteValue) serialisation. But for non-primitive it's enough to have only `objectId`, even without type. Should we send `{type: 'object', objectId: 'SOME_OBJECT_ID'}`, or we don'r require `type`?

For non-primitive objects I want to use as an argument, if I want to pass some new object as an argument, which was not presented in BiDi yet (like `{a: 1}`), I want to use be able to use BiDi serialisation protocol. But it requires making an `objectId` optional.

# UPD:
As discussed, command `script.callFunction` would be more readable.

**Comment by jgraham** [comment]
Yes, I was assuming we'd have something like `script.call[FunctionOn]`.

For arguments, we could indeed allow a serialization like `{objectId: <object id string>}` to refer to an object on the remote end. The question is whether to allow `type` and `value` fields. The advantage of that would be allowing the exact same object from a response to be used in an argument. The disadvantage is that it's unclear what to do if there's a mismatch (e.g. if you pass in `type: "object"` but once you lookup the object id get a `map`). 

I think I favour minimisng the amount of validation having a schema like:
```
Argument = (
   RemoteValueArgument //
   ValueArgument
)

RemoteValueArgument = {
    objectId: text
}

ValueArgument = {
    type: text,
    value: Argument
}
```

**Comment by sadym-chromium** [comment]
Merged in #142
